src/test.rs

```diff
@@ -153,10 +153,10 @@ impl TestContext {
         )
     }
 
-    /// Lower `hugr` to LLVM, then JIT and execute the function named `entry` in
-    /// the inner module.
+    /// Lower `hugr` to LLVM, then JIT and execute the function named
+    /// `entry_point` in the inner module.
```

Comment:
```suggestion
    /// by `entry_point` in the inner module.
```

I'm not quite sure what the "inner" "module" is (if AIUI we have a Hugr, not a Package, isn't that a/the-only module?), but nvm